types/jsdom/ts3.5/index.d.ts

```diff
@@ -52,7 +52,7 @@ declare module 'jsdom' {
     }
 
     class ResourceLoader {
-        fetch(url: string, options: FetchOptions): Promise<Buffer> | null;
+        fetch(url: string, options: FetchOptions): (Promise<Buffer> & { abort: () => void }) | null;
```

Comment:
It might be better to use an `AbortablePromise` interface, e.g.:
```ts
interface AbortablePromise<T> extends Promise<T> {
	abort(): void;
}
```

Reply:
Definitely! :smiley:
Fixed.